### DISCUSSION TITLE
Weather Notification app discussion

### DISCUSSION THREAD

**Discussion by diyatomar04** [Bhavyajain0604/weather-notification-#2] [General]
Hi Bhavya .
Lets first surf the net regarding our topic and then we can start working on it.

**Comment by Bhavyajain0604** [comment]
Sure diya I have find something which can help us regarding our project .
https://youtu.be/9P5MY_2i7K8?si=JC9jbJYGgqOTAyS7
https://youtu.be/HpbFZO3kCd0?si=ADgKEb9aVKd7WuGE
check this out once and let me know

**Comment by diyatomar04** [comment]
I have gone through this..I think we can go for 1st one..

**Comment by diyatomar04** [comment]
Lets have an offline meet and we can discuss the further details.

**Comment by Bhavyajain0604** [comment]
Okay diya we will have it tomorrow

**Comment by diyatomar04** [comment]
Cool..and about the visual representation that we have to give ..Should we go for kanban or should we prepare it on any other app??

**Comment by Bhavyajain0604** [comment]
I think as we discussed today we should go for kanabn only it will be more effective

**Comment by diyatomar04** [comment]
Okay. Alright!!
Lets start with Kanban first then we will move forward with coding part.

**Comment by Bhavyajain0604** [comment]
Yeah sure tomorrow we will setup kanban

**Comment by diyatomar04** [comment]
Bhavya i will draw a rough sketch of the weather notification . Can you suggest a little what should i add.

**Comment by diyatomar04** [comment]
Once i will draw it on a page , then we can present it visually.

**Comment by Bhavyajain0604** [comment]
Sure diya we can do like that i will suggest you what we should add there once it done on page then we can present it on kanban

**Comment by Bhavyajain0604** [comment]
Heyy Diya..let's have discussion tom after class..We have to see the basic design , color and all those things..

**Comment by diyatomar04** [comment]
Okay..after class it is then.

**Comment by diyatomar04** [comment]
Bhavya we don't have to give description of weather right since we just have to make a pop up kind of thing right ??

**Comment by Bhavyajain0604** [comment]
Yeah diya we have to do that only we will keep it as simple as we can otherwise it will create a lot of chaos we can add basic things like temperature, location, timing and some others things...

**Comment by diyatomar04** [comment]
Okay!!

**Comment by diyatomar04** [comment]
Then i think we should keep place name , time , current weather (symbol) and temperature . what do you say ?

**Comment by diyatomar04** [comment]
![20230930_025706](https://github.com/Bhavyajain0604/weather-notification-/assets/117175207/b5fa0128-a8eb-48ab-9f2e-516f684a5a9a)
I have made this rough sketch ..
We can make our pop up look similar to this.

**Comment by Bhavyajain0604** [comment]
Hi diya could you please upload a user case we had discussed on that day..

**Comment by diyatomar04** [comment]
Yes sure.
And Bhavya since you have made the account for generating the API key , mail the website for same.

**Comment by diyatomar04** [comment]
![20231006_093731](https://github.com/Bhavyajain0604/weather-notification-/assets/117175207/cff7073e-1235-4e49-9e77-73650b8ceba9)
![Uploading 20231006_093721.jpg…]()

**Comment by diyatomar04** [comment]
Lets start pushing the code in the repository.

**Comment by diyatomar04** [comment]
And Bhavya please once check the API key generation

**Comment by diyatomar04** [comment]
https://forms.gle/yES8wty8p3T6zkDc7 Feedback form
